lib/maverick/api.ex

```diff
@@ -58,6 +60,23 @@ defmodule Maverick.Api do
       end
 
       def list_routes(), do: Maverick.Route.list_routes(@otp_app, @root_scope)
+
+      def router() do
+        @router
+      end
+
+      def init(opts) do
+        Maverick.Api.Initializer.init(__MODULE__)
+        apply(@router, :init, [opts])
```

Comment:
is this ever called? Since we're building the router module for the user, we don't really give them an opportunity to define this `init` function, right?

Reply:
it is called by the plug_adapter, in our case this is currently plug_cowboy